checks/checks.md

```diff
@@ -104,7 +120,9 @@ This check tries to determine if the project requires pull requests for all chan
 
 ## SAST 
 
-This check tries to determine if the project uses static code analysis systems. It currently works by looking for well-known results in GitHub pull requests. More specifically, the check first looks for Github apps named [github-code-scanning](https://securitylab.github.com/tools/codeql) (codeql) and  sonarcloud in the recent (~30) merged PRs. If >75% of commits contain at least a successful check (by any of the apps above), the check succeeds. If the above fails, the check instead looks for the use of "github/codeql-action" in a github workflow. 
+This check tries to determine if the project uses static code analysis systems. 
+SAST tool may prevent known classes of bugs to be inadvertently  introduced in the codebase. A a low score is considered `Medium` risk.
```

Comment:
s/A a/a

Reply:
also, we should check and fix some double spaces in this yaml file, e.g. "inadvertently  introduced"